src/inspector/models/notation/notationsettingsproxymodel.cpp

```diff
@@ -21,81 +21,71 @@
-    addModel(new BarlineSettingsModel(this, repository));
-    addModel(new StaffSettingsModel(this, repository));
-    addModel(new SectionBreakSettingsModel(this, repository));
-    addModel(new MarkerSettingsModel(this, repository));
-    addModel(new JumpSettingsModel(this, repository));
-    addModel(new KeySignatureSettingsModel(this, repository));
-    addModel(new AccidentalSettingsModel(this, repository));
-    addModel(new FretDiagramSettingsModel(this, repository));
-    addModel(new PedalSettingsModel(this, repository));
-    addModel(new SpacerSettingsModel(this, repository));
-    addModel(new ClefSettingsModel(this, repository));
-    addModel(new HairpinSettingsModel(this, repository));
-    addModel(new CrescendoSettingsModel(this, repository));
-    addModel(new StaffTypeSettingsModel(this, repository));
-    addModel(new TextFrameSettingsModel(this, repository));
-    addModel(new VerticalFrameSettingsModel(this, repository));
-    addModel(new HorizontalFrameSettingsModel(this, repository));
-    addModel(new ArticulationSettingsModel(this, repository));
-    addModel(new OrnamentSettingsModel(this, repository));
-    addModel(new AmbitusSettingsModel(this, repository));
-    addModel(new ImageSettingsModel(this, repository));
-    addModel(new ChordSymbolSettingsModel(this, repository));
-    addModel(new BracketSettingsModel(this, repository));
-    addModel(new BraceSettingsModel(this, repository));
-    addModel(new TimeSignatureSettingsModel(this, repository));
-    addModel(new BendSettingsModel(this, repository));
-    addModel(new TremoloBarSettingsModel(this, repository));
-    addModel(new MMRestSettingsModel(this, repository));
-    addModel(new TremoloSettingsModel(this, repository));
-    addModel(new MeasureRepeatSettingsModel(this, repository));
+    AbstractInspectorModel::InspectorModelType notationSingleElementModelType = this->notationSingleElementModelType(elementSet);
+
+    if (notationSingleElementModelType != AbstractInspectorModel::InspectorModelType::TYPE_UNDEFINED) {
+        auto model = inspectorModelCreator()->newInspectorModel(notationSingleElementModelType, parent, repository);
+        setTitle(model->title());
+        addModel(model);
+    } else {
+        setTitle(qtrc("inspector", "Notation"));
+        for (const Ms::ElementType elementType : elementSet) {
+            AbstractInspectorModel::InspectorSectionType sectionType = AbstractInspectorModel::sectionTypeFromElementType(elementType);
+            if (sectionType == AbstractInspectorModel::InspectorSectionType::SECTION_UNDEFINED) {
+                continue;
+            }
+
+            AbstractInspectorModel::InspectorModelType modelType = AbstractInspectorModel::notationElementModelType(elementType);
+            addModel(inspectorModelCreator()->newInspectorModel(modelType, parent, repository));
+        }
+    }
+}
+
+bool NotationSettingsProxyModel::canContains(Ms::ElementType elementType) const
```

Comment:
judging by the logic it makes sense to call it like "hasSupportOfType"

Reply:
fixed